search-scores-and-weights.md

```diff
@@ -0,0 +1,40 @@
+Our search system uses scoring to assign values to each document based on
```

Comment:
Maybe add a Title to the document and a Date of creation. And if you have it, the current weight.